### PR TITLE
[MM-61077] Fix errcheck issues in server/channels/web/static.go

### PR DESCRIPTION
**Pull request by theoforger**
#### Summary
This PR fixes errcheck issues in [`server/channels/web/static.go`](https://github.com/mattermost/mattermost/blob/master/server/channels/web/static.go)

#### Ticket Link
Fixes https://github.com/mattermost/mattermost/issues/28749
Jira https://mattermost.atlassian.net/browse/MM-61077

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
LGTM :+1:

File: server/channels/web/static.go [hunk 1]
```diff
@@ -78,6 +83,9 @@ func root(c *Context, w http.ResponseWriter, r *http.Request) {
 	staticDir, _ := fileutils.FindDir(model.ClientDir)
 	contents, err := os.ReadFile(filepath.Join(staticDir, "root.html"))
 	if err != nil {
+		c.Logger.Warn("Failed to read content from file",
+			mlog.String("file_path", filepath.Join(staticDir, "root.html")),
+			mlog.Err(err))
 		http.Error(w, err.Error(), http.StatusNotFound)
```
Comment (streamer45) [inline server/channels/web/static.go:89]: For consistency, I think this should be an internal error rather than not found.

**Review by streamer45 — APPROVED** [review]
Nice work!

**Comment by theoforger** [comment]
The requested change is applied. Thanks for the reviews 🙏!

**Review by hanzei — APPROVED** [review]

**Pull request merged**